docs/installation.rst

```diff
@@ -4,97 +4,99 @@
 Installation
 ************
 
-Requirements
-============
+The SunPy project `maintains a range of packages <https://sunpy.org/project/affiliated>`__ that leverage the wider ecosystem of scientific Python packages for solar physics.
 
-sunpy requires Python 3.8 or higher.
+Python distribution
+===================
+These instructions will set you up with `miniforge <https://conda-forge.org/docs/user/introduction.html>`__, which makes it easy to install and manage Python packages.
 
-Installing Scientific Python and sunpy
-======================================
+`To install the miniforge Python distribution follow these instructions <https://github.com/conda-forge/miniforge#install>`__.
 
-sunpy is part of the wider ecosystem of scientific Python packages for solar physics.
-Therefore a working sunpy installation is more about installing the scientific Python ecosystem than sunpy itself.
+Installing miniforge will set the default conda channel to ``conda-forge``, which is required for installing ``sunpy`` and many other packages in the scientific Python ecosystem.
 
-If you do not currently have a working scientific Python distribution this guide will set you up with the Miniconda, which makes it easy to install and manage your scientific Python packages.
+Installing a package using miniforge
+------------------------------------
+To install ``sunpy``, launch a terminal (under a UNIX-like system) or miniforge Prompt (under Windows).
 
-To install the Miniconda Python distribution follow the instructions at
-`here <https://docs.conda.io/en/latest/miniconda.html>`__.
-Although Miniconda makes it simple to switch between Python versions, we recommend that new users install the latest Python 3.x version of Miniconda.
-
-The reason we choose Miniconda over Anaconda, is mainly due to the size as Anaconda comes with a full install of packages you probably do not need and this way you have more direct control over what has been installed into your Python virtual environment.
-Furthermore, you bypass the need for the conda resolver to sort out your root environment which should make conda faster to use.
-
-Installing sunpy using Miniconda
---------------------------------
+.. note::
 
-To install sunpy launch a system command prompt or the 'Anaconda Prompt' (under Windows).
-First configure conda for to add the `conda-forge channel <https://conda-forge.org/>`__::
+    We strongly recommend using a `Python virtual environment <https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/>`__ or a `conda virtual environment. <https://towardsdatascience.com/getting-started-with-python-environments-using-conda-32e9f2779307>`__
```

Comment:
The instructions have just stepped through installing `miniforge`, so to avoid confusion I think this note should be removed?

Reply:
You mean our instructions? Installing miniforge doesn't mean they are using virtual environments correctly, they could be installing into base.

Reply:
Yeah, our instructions have just stepped through installing `miniforge`. Guess that's right that they still need to get a conda env working.